### DISCUSSION TITLE
Issue Tracker Feedback

### DISCUSSION THREAD

**Discussion by gnalvesteffer** [Teardown-Issue-Tracker-Maintainers/Teardown-Issue-Tracker#2] [General]
Welcome to the Teardown Issue Tracker repo! In collaboration with the Teardown devs, I've set up this repo for the Teardown community to provide a uniform way of reporting bugs/feature requests/questions to the devs and other more technically savvy people.

If you have any feedback/suggestions to improve this issue tracker, such as new issue templates, or changes to an existing issue template, please let me know!

**Comment by carsakiller** [comment]
Perhaps adding a "System Info" section to the templates would be helpful for the devs. Somewhere for people to detail what their system is running:
- GPU
- Graphics driver version
- CPU
- OS
- Teardown Version

**Reply by gnalvesteffer** [reply]
I have a similar section for this in the bug report template but I can update it to include these other fields 👍

**Reply by gnalvesteffer** [reply]
I've updated the bug report issue template to have these additional fields @carsakiller.

Commit: https://github.com/Teardown-Issue-Tracker-Maintainers/Teardown-Issue-Tracker/commit/bc3bf9c0a8d4e58a99d56338f1d7d105074b3c42

**Comment by UwQ-Official** [comment]
also, is there a typo or am I dumb (*supposed* instead of *suppose*) [On **Expected behavior**, Bug report template]
![Screenshot 2025-06-28 231748](https://github.com/user-attachments/assets/3fbe2c4a-2484-4f41-8920-e98052b2f651)
